src/Model/Fulfilment/Order.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\src\Model\Fulfilment;
+
+use DateTime;
+use MyParcelNL\Sdk\src\Adapter\DeliveryOptions\DeliveryOptionsFromOrderAdapter;
+use MyParcelNL\Sdk\src\Model\Recipient;
+use MyParcelNL\Sdk\src\Support\Collection;
+
+class Order extends AbstractOrder
+{
+    /**
+     * @param  array $data
+     *
+     * @throws \Exception
+     */
+    public function __construct(array $data = [])
+    {
+        $this->uuid                          = $data['uuid'] ?? null;
+        $this->external_identifier           = $data['external_identifier'] ?? null;
```

Comment:
Hier ook de snake_case: ik vind dat zelf ook intuitief correct, maar wanneer mag het en wanneer niet?

Reply:
Dit doen we ook in andere bestanden (en in de api), is wmb dan gewoon een standaard. Vind het ook wel weer wat hebben zo.

Reply:
Alleen bij properties dan